src/v2/view-builder/views/SSOExtensionView.js

```diff
@@ -14,7 +14,17 @@ const Body = BaseForm.extend({
 
   initialize () {
     BaseForm.prototype.initialize.apply(this, arguments);
-    Util.redirectWithFormGet(this.options.currentViewState.href);
+
+    // TODO: this should be handle by the foundation,
+    // not on the view level
+    const method = this.options.appState.get('rawIdxState').remediation.value
```

Comment:
Can you see if you use this.options.appState.get('remediations')[0].value here instead of looking at the `rawIdxState`?

Reply:
`remediations` does not contain the method info.

Reply:
I am trying to see if we can avoid looking into `rawIdxState` from within our views. The job of the transformers is to make sure views have the data they need. Can we try to see if we can `method` to https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/responseTransformer.js#L71  so that its available for the view ?